crayons.py

```diff
@@ -147,3 +148,11 @@ def enable():
     global DISABLE_COLOR
 
     DISABLE_COLOR = False
+
+
+def replace_colors(replace_dict):
+    """Replace colors to customize the look under specific background."""
+    global REPLACE_COLORS
+
+    assert isinstance(replace_colors, dict)
+    REPLACE_COLORS = {k.upper(): v.upper() for k, v in replace_colors.items()}
```

Comment:
I'm guessing you meant `replace_dict` instead of `replace_colors` for these two lines (as the assert line fails here).